src/CourseDashboard.cpp

```diff
@@ -1,4 +1,123 @@
 #include "CourseDashboard.hpp"
+#include "Student.hpp"
 #include <stdexcept>
+#include <string>
+#include <iostream>
+#include <fstream>
+
 
 CourseDashboard::CourseDashboard() {}
+void CourseDashboard::addStudent(Student &user){
+    std::vector<std::string> student{};
+    student.push_back(user.getNick());
+    student.push_back(user.getName());
+    student.push_back(user.getGitHub());
+    student.push_back(user.getFirecode());
+ 
+    std::fstream baza;
+    baza.open("baza.txt", std::ios::out|std::ios::app);
+    for(unsigned int i = 0; i < student.size(); i++)
+        baza << student[i] << std::endl;
+    baza.close();
```

Comment:
Writing to file was not needed for US1.
The expectation was to keep students in some container. But it's good that you have used fstream.